### PR TITLE
feat: add bundle job to circleci config

### PR DESCRIPTION
**Pull request by bmo-tv**
**Summary of change:**

This is a simple addition to the `./circleci/.config.yml` file to include a `bundle` job. This `bundle` job is responsible for creating a `.zip` artifact using the already provided `npm run zip` command.

**Have the changes in this PR been added to the documentation for this project?**

Does not apply

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -20,3 +34,7 @@ workflows:
   check-standards:
     jobs:
       - standards
+      - bundle:
+          filters:
```
Comment (bmo-tv) [inline .circleci/config.yml]: What are y'alls thoughts on how frequently this build should run? Should we limit it to specific branches?

Comment (nickcernis) [reply .circleci/config.yml]: @bmo-tv Thanks for this! Filtering on tags seems right, the same as [AB Pro](https://github.com/studiopress/atomic-blocks-pro/blob/46effb3b9e7be1f9fdd7b22562e78f70ae0aedcf/.circleci/config.yml#L140).

Only tagging of the master branch should trigger a zip and deploy if you wanted to restrict by branch, although we do not tend to tag other branches.

Comment (bmo-tv) [reply .circleci/config.yml]: Updated the filters to isolate this job to only the `master` branch

Comment (nickcernis) [reply .circleci/config.yml]: @bmo-tv Thanks! Looks good.

**Pull request merged**